src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -27,13 +28,15 @@ public class CompletedDownloadService : ICompletedDownloadService
         private readonly IDownloadedEpisodesImportService _downloadedEpisodesImportService;
         private readonly IParsingService _parsingService;
         private readonly Logger _logger;
+        private readonly ISeriesService _seriesService;
 
         public CompletedDownloadService(IConfigService configService,
                                         IEventAggregator eventAggregator,
                                         IHistoryService historyService,
                                         IDownloadedEpisodesImportService downloadedEpisodesImportService,
                                         IParsingService parsingService,
-                                        Logger logger)
+                                        Logger logger,
+                                        ISeriesService seriesService)
```

Comment:
pretty minor, but we usually keep logger as the last arguments, (or at least try, you don't have to fix it but just for future)